filebeat/module/elasticsearch/deprecation/_meta/fields.yml

```diff
@@ -2,3 +2,15 @@
   type: group
   description: >
   fields:
+    - name: data_stream.dataset
```

Comment:
@klacabane Are all theses fields really problematic in the test suite, I tough  they would have been unecessary. Can you paste all the failures?

Reply:
No, we have to keep them with the new indexing strategy.  The problem I see here is we have _a hybrid approach_ issue. I need to check a few things.

Reply:
OK, I've double checked with the elasticsearch's default template and the types here are the same.